schemas/epigenomics/examples/activities.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "ChIP_6802",
+    "type": "ChIP",
+    "used": ["BioSAli_6802"],
+    "generated": ["Library_9367"]
+}
+{
+    "id": "Sequencing_CoPA_13999",
+    "type": "Sequencing",
+    "used": ["Library_9367"],
+    "generated": [
+        "SequenceFile_CoPA_13999_1",
+        "SequenceFile_CoPA_13999_2"
+    ],
+    "informed": "ChIPSeq_6802"
```

Comment:
Remove this?  I thought we were getting rid of ChIPSeq

Reply:
Ah, good catch